src/azul/service/manifest_service.py

```diff
@@ -175,7 +187,7 @@ def _derive_manifest_key(self, filters: Filters) -> str:
         git_commit = config.lambda_git_status['commit']
         manifest_namespace = uuid.UUID('ca1df635-b42c-4671-9322-b0a7209f0235')
         filter_string = repr(sort_frozen(freeze(filters)))
-        return str(uuid.uuid5(manifest_namespace, git_commit + filter_string))
+        return str(uuid.uuid5(manifest_namespace, git_commit + filter_string + format_ + str(content_hash)))
```

Comment:
We should use a separator between fields, as in

```suggestion
        content_hash = str(content_hash)
        return str(uuid.uuid5(manifest_namespace, ','.join(git_commit, format_, content_hash, filter_string)))
```
This is a way to avoid collisions as in 

"a" + "ab" == "aa" + "b"

Since commas *can* occur in the filter string, it should go last. We should also assert that commas don't occur in the other fields.